lib/webidl2.js

```diff
@@ -146,12 +153,20 @@
       if (!tokens.length || tokens[0].type !== type) return;
       if (typeof value === "undefined" || tokens[0].value === value) {
         last_token = tokens.shift();
+        line += count(last_token.trivia, "\n");
         if (type === ID && last_token.value.startsWith('_'))
           last_token.value = last_token.value.substring(1);
         return last_token;
       }
     }
 
+    function unconsume(...toks) {
+      for (let tok of toks) {
```

Comment:
Nit: const tok

Reply:
Oops, I forgot adding a comment, this was done because of Servo compatibility (was introduced in #139 and removed in #150).

Reply:
ah yeah. Ok, cool.